### DISCUSSION TITLE
GTN Outreach discussion

### DISCUSSION THREAD

**Discussion by bebatut** [galaxyproject/training-material#980] [General]
This discussion was raised during the last GTN meeting #954

@tnabtaf what do you think?

**Comment by tnabtaf** [comment]
The ideas discussed in #979 are good:

- [x] landing pages for Usegalaxy.*
- [x] Monthly news and Twitter

I would also add

- Highlight on galaxyproject.org landing page.
- Highlight big recent tutorial changes/updates in the [Hub and Doc updates](https://galaxyproject.org/news/2018-09-galaxy-update/#hub-and-doc-updates) section of the monthly news.  (That section is only 2 months old.)
- ~Push the tutorial of the month (see #979) to the GOBLET directory that month as well.  (Have to start somewhere with GOBLET.)~
